.github/workflows/mutation-testing-annotations.yaml

```diff
@@ -0,0 +1,57 @@
+# yamllint disable rule:line-length
+# yamllint disable rule:braces
+
+name: Infection
+
+on:
+    pull_request:
+    push:
+        branches:
+            - main
+            - master
```

Comment:
This workflow for PRs only so this `push:` section can go. This will also simplify `get_infection_pr_flags()`

Reply:
Frankly there will be only one line left, so it'll be as good as it is inlined.